File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/AspPageBox/Functionality/EmbedASPNETPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.AspPageBox.Functionality
{
    public partial class EmbedASPNETPage : UserControl
    {
        public EmbedASPNETPage()
        {
            InitializeComponent();
        }

        private void EmbedASPNETPage_Load(object sender, EventArgs e)
        {
            this.demoAspPageBox.Path = @"Controls\AspPageBox\Functionality\AspViewPage.aspx";
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/AspPageBox/Functionality/EmbedASPNETPage.vb
Namespace CompanionKit.Controls.AspPageBox.Functionality

	Public Class EmbedASPNETPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

	        

		End Sub

        Private Sub EmbedASPNETPage_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
            Me.demoAspPageBox.Path = "Controls\AspPageBox\Functionality\AspViewPageVB.aspx"
        End Sub
    End Class

End Namespace